testsuite/launch.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       containers:
       - name: azure-arc-kubernetes-conformance
         # test image will change to actual once we push to MCR
-        image: testsantoshacr.azurecr.io/arck8sconformance/launcher:0.0.37
+        image: testsantoshacr.azurecr.io/arck8sconformance/launcher:0.0.78
```

Comment:
Why is this change required?

Reply:
the container version 0.0.78 is having the latest change of launcher container code. previous versions are giving error. Eventually we have to change this ACR path to MCR path when we push the docker image.